src/main/java/com/twilio/oai/api/ApiResourceBuilder.java

```diff
@@ -113,9 +116,12 @@ public ApiResources build() {
     }
 
     protected Map<String, Object> mapOperation(CodegenOperation operation) {
-        Map<String, Object> operationMap = new HashMap<>();
+        final String operationType = Utility.populateCrudOperations(operation);
+        metaAPIProperties.put(operationType, operation);
+
+        Map<String, Object> operationMap = operation.vendorExtensions;
         final HttpMethod httpMethod = HttpMethod.fromString(operation.httpMethod);
-        if (isInstanceOperation(operation)) {
+        if (PathUtils.isInstanceOperation(operation)) {
```

Comment:
Previously, we had set only Fetch, Update and Delete operations as instance operations, now since this is changed we can have Create as instance operation and this is affecting the operation name

Reply:
Create and Read (List) should generally be list operations. The other 3 are instance operations.

Reply:
Depends on the pathType, if the pathType is "instance", all the operations under the path are treated as instance operations even if it is Create

Reply:
example: [CreateGooddata](https://github.com/twilio/twilio-oai/blob/main/spec/yaml/twilio_flex_v1.yaml#L1270)

Reply:
Yes, there are exceptions, but this is already handled by by Node: https://github.com/twilio/twilio-node/blob/4de330032482d6cf2e079e92a2e75adc503537e8/lib/rest/flexApi/v1/goodData.ts#L30

Reply:
In PHP, we have treated all the instance operations equally and hence the generated code will show incorrect function name. How about setting the "x-name" independently of checking whether the operation is of type instance or list ?

Reply:
Works for me. I've updated it to use the logic used by Node.